### PR TITLE
Workflows GUI

### PR DESCRIPTION
**Pull request by antgonza**
This is the initial changes to allow (use?) the DefaultWorkflows in Qiita. To start the easiest is to reuse and create a GUI to visualize the current DefaultWorkflows. A few changes here:
- We don't need to have command_id in DefaultWorkflowNode because we can get the command from the DefaultParameters
- Adding an active flag to default_workflow
- During the development of the GUI, I realized that we are missing 2 map files (so the GUI wasn't working as expected)

The new GUI looks like this:
workflows

### CODE REVIEW

**Comment by antgonza** [comment]
This new version looks like this: 
Screen Shot 2021-02-24 at 8 50 44 AM

**Review by ElDeveloper — APPROVED** [review]
Looks great, just a few small suggestions!

File: qiita_pet/templates/workflows.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    dragNodes: false,
+    dragView: true,
+    zoomView: true,
+    selectConnectedEdges: true,
+    navigationButtons: true,
+    keyboard: false,
+  }
+};
+
+var colorScheme =  {
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_pet/templates/workflows.html:45]: I think this is fine for the full page.

**Comment by ElDeveloper** [comment]
Thanks @antgonza

**Pull request merged**